assignments/a6.md

```diff
@@ -51,7 +56,33 @@ It is suggested to use a folder and multiple files, just like this CSC491 repo,
 
 # Rubric
  
-**TODO**
+| Area | Description| Weight |
+| --- | --- | --- |
+| Applications | All applications present in the architecture diagram have a repository and an MVP is implemented | 20.0 |
+| Test coverage | Tests should cover at least 65% of your application, upload to the DCSIL Team App, and upload to Code Climate. | 20.0 |
+| Code Quality | All repositories have at least a "B" Grade on Code Climate | 10.0 |
+| Operational Excellence | CI, Exceptions, Logging, and all other services listed in the requirements & `service.yml` example are all setup. Bootstrap script works. Service.yml is uploaded with no errors in the DCSIL team app | 10.0 |
+| Dynamic | The majority of content is functional and dynamic. The apps do not rely on hard coded static data | 10.0 |
+| Use Cases | All main use cases/flows have been delivered and are functional. | 15.0 |
+| Video | A short video with a short write up (some bullet points explaining the video) has been provided that indicates the flows that have been implemented. This allows the teaching staff to mark and is often highly valued in a production company. Please note any areas which are still under development, or not fully implemented. Keep in mind the majority of the paths _should_ be implemented. | 5.0 |
+| Design | The application has a consistent look and feel. You didn't rely on default styles (see below). The flows are not incoherent. This will be a highly subjective area, so feel free to ask for advice before submission (please don't ask the day before, or day of, submission). | 10.0 |
```

Comment:
I am pleased to hear that you are considering design as one of the factors in their grade. Since this is subjective, maybe a little more clarity would be helpful here - such as objective criteria. Something along the lines of:
* Navigation is simple, consistent, and easy so that someone who has never used this website can get around with ease.
* Typeface is readable and accessible.
* Actionable buttons behave intuitively and warn users when performing a destruction action.
* The website should allow users to use browser back and forward (history) buttons, without breaking the state of the website.  

It might be worth pointing to [GitHub Primer's Interface Guidelines](https://primer.style/design/) as a reference, although it's doesn't cover very much.

Reply:
This is definitely valid. I am worried it might take a long time to grade. I'll have a think on this!

Reply:
I'll include it as a guide I think, but I'm not sure I can quantify it as a grade, unfortunately